dist/node-connector.js

```diff
@@ -16676,6 +16676,18 @@ module.exports =
 
 	    _classCallCheck(this, MapdCon);
 
+	    this.connectAsync = function () {
```

Comment:
maybe it there are multiple methods wrapped in a premise you could write a promiseWrapper like it was done by @jrajav on a recent PR I could link to.

Reply:
Ah, you are mentioning it in the description :+1:

Reply:
yeah @jrajav and I discussed that we can refactor this after his PR #70 pulls in these changes from master. 

There are many places in this library where an `*async` method wraps another method that uses a callback with a Promise, so what I'm doing here is nothing new in that regard. This is more of an attempt to start standardizing the methods so that we can use Promises consistently through out.